src/context/QRContext.tsx

```
@@ -35,76 +44,180 @@ const getSafeLocalStorage = () => {
+        // Fallback for when worker is not available (e.g., enterprise security policies)
+        if (name === 'scannability-fail' && state.preferences.telemetryOptIn) {
+          try {
+            fetch('/api/telemetry/scannability', {
+              method: 'POST',
+              headers: { 'Content-Type': 'application/json' },
+              body: JSON.stringify({
+                engine: detail?.engine,
+                styleId: detail?.styleId || state.config.style || 'default',
+                errorType: detail?.errorType,
+                timestamp: Date.now()
+              }),
```

ESLint

[error] Direct use of JSON.stringify is restricted for security reasons. Consider using safeJsonLdStringify for JSON-LD schemas or review carefully to prevent XSS.

(no-restricted-syntax)